azurelinuxagent/ga/update.py

```diff
@@ -251,6 +251,15 @@ def run(self, debug=False):
             logger.info(u"Agent {0} is running as the goal state agent",
                         CURRENT_AGENT)
 
+            # Log OS-specific info, locally and as a telemetry event.
+            msg = u"Distro info: {0}:{1}, osutil class being used: {2}, " \
+                  u"agent service name: {3}".format(DISTRO_NAME, DISTRO_VERSION,
+                                                    type(self.osutil).__name__, self.osutil.service_name)
+            add_event(AGENT_NAME,
```

Comment:
don't our telemetry events already have the OS info?

Reply:
Yes - It is added in the [init_sysinfo](https://github.com/Azure/WALinuxAgent/blob/0105f5f4d45e051fdfc1bb78eeccb0d14967bcc6/azurelinuxagent/ga/monitor.py#L163).

Reply:
Correct. The purpose of this event was to assert that the correct `osutil` was created based on distro info.